deploy/templates/service-template.yml

```diff
@@ -0,0 +1,146 @@
+---
+kind: Template
+apiVersion: template.openshift.io/v1
+metadata:
+  name: assisted-migration-service
+parameters:
+  - name: MIGRATION_PLANNER_IMAGE
+    required: true
+  - name: IMAGE_TAG
+    value: latest
+  - name: MIGRATION_PLANNER_REPLICAS
+    description: Number of replicas of the service to run.
+    value: "3"
+  - name: MIGRATION_PLANNER_MEMORY_REQUEST
+    description: Memory request for the API pods.
+    value: "400Mi"
+  - name: MIGRATION_PLANNER_MEMORY_LIMIT
+    description: Memory limit for the API pods.
+    value: "2000Mi"
+  - name: MIGRATION_PLANNER_CPU_REQUEST
+    description: CPU request for the API pods.
+    value: "300m"
+  - name: MIGRATION_PLANNER_CPU_LIMIT
+    description: CPU limit for the API pods.
+    value: "500m"
+  - name: MIGRATION_PLANNER_AGENT_IMAGE
+    description: Migration Discovery Service image
+    required: true
+  - name: MIGRATION_PLANNER_URL
+    description: The API URL of the migration assessment
+    required: true
+  - name: MIGRATION_PLANNER_UI_URL
+    description: The console URL of the migration assessment
+    required: true
+  - name: DB_SECRET_NAME
+    description: The name of the OpenShift Secret used for the database.
+    displayName: Database Secret Name
+    value: migration-planner-db
+
+objects:
+  - kind: ServiceAccount
+    apiVersion: v1
+    metadata:
+      name: migration-planner
+  - kind: Service
+    apiVersion: v1
+    metadata:
+      labels:
+        app: migration-planner
+      name: migration-planner
+    spec:
+      ports:
+        - name: migration-planner
+          port: 3443
+          protocol: TCP
+          targetPort: 3443
+      selector:
+        app: migration-planner
+  - kind: Service
+    apiVersion: v1
+    metadata:
+      labels:
+        app: migration-planner
+      name: migration-planner-agent
+    spec:
+      ports:
+        - name: migration-planner-agent
+          port: 7443
+          protocol: TCP
+          targetPort: 7443
+      selector:
+        app: migration-planner
+  - kind: Deployment
+    apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
+    metadata:
+      name: migration-planner
+    spec:
+      selector:
+        matchLabels:
+          app: migration-planner
+      replicas: ${{MIGRATION_PLANNER_REPLICAS}}
+      template:
+        metadata:
+          labels:
+            app: migration-planner
+        spec:
+          containers:
+            - name: migration-planner
+              resources:
+                requests:
+                  cpu: ${MIGRATION_PLANNER_CPU_REQUEST}
+                  memory: ${MIGRATION_PLANNER_MEMORY_REQUEST}
+                limits:
+                  cpu: ${MIGRATION_PLANNER_CPU_LIMIT}
+                  memory: ${MIGRATION_PLANNER_MEMORY_LIMIT}
+              image: ${MIGRATION_PLANNER_IMAGE}:${IMAGE_TAG}
+              imagePullPolicy: Always
+              ports:
+                - containerPort: 3443
```

Comment:
7443 it's missing.

Reply:
7443 is exposed throw the service, do you think it need to be declared here as well?

Reply:
for consistency only.  it does not change anything not exposing it here

Reply:
The port will be exposed anyway, but as per documentation it's a good practise:
https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#ports

Reply:
I have added the port